views/admin/taxonomy-term/list.php

```diff
@@ -1,5 +1,10 @@
-<?php echo head(array('title' => __('%s taxonomy terms', $taxonomy->name))); ?>
-<?php echo flash(); ?>
+<?php
+$total = count($terms);
+echo head(array(
+    'title' => __('%s taxonomy terms (%s)',
+        $taxonomy->name, empty($total) ? __('none') : ($total == 1 ? __('%d term', $total) : __('%d terms', $total))),
```

Comment:
Use __(plural('one term', '%s terms', $total)) instead. It improves translatability for languages where plural forms are handled differently